spec/controllers/sessions_controller_spec.rb

```diff
@@ -4,11 +4,23 @@
   it { should be_a Clearance::BaseController }
 
   describe 'on GET to /sessions/new' do
-    before { get :new }
+    context 'when a user is not signed in' do
```

Comment:
oh it has been... but the whole file was already in single quotes so I thought that is what the project was using... I will change all to double now.